src/smac_planner_hybrid.cpp

```diff
@@ -378,9 +490,10 @@ uint32_t SmacPlannerHybrid::makePlan(
   std::cout << "It took " << (c - b).toSec() * 1000 <<
     " milliseconds to smooth path." << std::endl;
 #endif
-
-  plan = std::move(output_path.poses);
-  return mbf_msgs::GetPathResult::SUCCESS;
+  plan_result.path = std::move(output_path.poses);
+  plan_result.length = Utils::length(plan_result.path);
```

Comment:
F: add this to the struct and add `getter` and `setter` for the path
length should be calculated in the `setter`